lib/base.js

```diff
@@ -38,6 +38,24 @@ class DocToolsBase extends Command {
     argv.baseDir = baseDir = this.baseDir = path.resolve(baseDir);
     // $baseDir/run/doctools
     argv.targetDir = this.targetDir = path.join(argv.baseDir, 'run/doctools');
+
+    // If we've already had this folder, remove it
+    // 1. There're some files useless and they will be left if we don't remove the folder
+    // 2. Sometimes when we remove some folders in the 'run/doctools' and we directly run
+    // 'npm run test', the original files won't be removed this will make us make mistakes
+    // and we won't remove the related links, this will cause the submit error and the local
+    // test cannot be checked out.
+    const removePath = path.join(argv.baseDir, 'run');
+    try {
+      yield rimraf(removePath);
+      this.logger.info(`'${removePath}' is successfully removed, preparing to re-generate folders and files`);
+    } catch (e) {
+      this.logger.warn(`'${removePath}' failed to be removed, 
```

Comment:
most time, we don't need to catch `rimraf`.

Reply:
In order to avoid making mistakes, this is a safer way to notify you ;)